thirty-five-mm/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user, :logged_in?
+
+  def current_user
```

Comment:
Your current_user checks for session[:user] but your login sets session[:user_id]